common/src/main/java/com/wynntils/screens/questbook/WynntilsQuestBookScreen.java

```diff
@@ -432,4 +406,59 @@ public void setContentSortOrder(ContentSortOrder newSortOrder) {
         setQuests(getSortedQuests());
         this.setCurrentPage(0);
     }
+
+    private void addQuestProgressTooltipLines(List<Component> tooltipLines, boolean miniQuestMode) {
```

Comment:
Something very similar to this information is now available as tooltips from Wynncraft. See e.g. the call chain around DiscoveryModel.updateWorldDiscoveriesFromQuery() on how go extract it.

I'm not sure if it is a suitable replacement, but maybe. (Otoh, changing this can be the fodder of another PR)

Reply:
If you don't mind, let's keep it like this now. (I also believe we could add some custom progress info with this code..)

Reply:
Yes, it was more a case of "oh, this reminds me of...".